packages/fxa-content-server/server/lib/routes/get-index.js

```diff
@@ -51,6 +52,7 @@ module.exports = function (config) {
     scopedKeysEnabled: SCOPED_KEYS_ENABLED,
     scopedKeysValidation: SCOPED_KEYS_VALIDATION,
     staticResourceUrl: STATIC_RESOURCE_URL,
+    subscriptions: SUBSCRIPTIONS,
```

Comment:
Please [update the unit test](https://github.com/mozilla/fxa/blob/68978f34c9ed96536a6adc490f6d2f6a178d34b6/packages/fxa-content-server/tests/server/routes/get-index.js#L79) to ensure this field and the expected values it contains are sent.